.linting/duplicate-ids-whitelist.txt

```diff
@@ -297,6 +297,7 @@ ec2InternalHostname 2
 ec2Region 2
 empty 5
 enqueue 4
+ensureReAuthorised 2
```

Comment:
Someone snuck a new duplicate thru on a different branch before the check was in place, so we need to update this here for now; This problem should go away now that all PRs are checked.

Reply:
I guess that someone was me.